lab3/euler.cpp

```diff
@@ -0,0 +1,44 @@
+#include <fstream>
+
+int TIMENUMBERS  = 100000;
+double x_0 = 5;
+double v_0 = 0;
+double dt = 0.01;
+double omega = 0.5;
+
+void output(double *x, double *x1, double *v, double *v1, double *t)
+{
+    std::ofstream foute;
+    foute.open("points.txt", std::ios::app);
+    foute << *x << '\t' << *x1 << '\t' << *v << '\t' << *v1 << '\t' << *t << std::endl;
+    foute.close();
+}
+
+
+
+int main()
```

Comment:
Just to make sure, you know how to pass command line arguments to C/CPP programs?

Reply:
In a way, yes.